metatron-app/metatron-discovery
# #3767 improved address copy and optional embedded dashboard

### Description
improved address copy and optional embedded dashboard

**Related Issue** : #3767,  #3769  


### How Has This Been Tested?

1. Check if you can copy the address of the dashboard and embedded dashboard from the top right menu in the workbook.
2. Make sure you can copy the chart's address from the chart's info layer.
3. Check if the top area disappears when the selectionFilter=false parameter is added in the embedded dashboard.
4. Check if the refresh function in the upper area disappears when the autoOn=false parameter is added in the embedded dashboard.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
